packages/commons-test/src/testUtils.ts

```diff
@@ -173,3 +175,14 @@ export const getMockPurposeVersion = (): PurposeVersion => ({
   dailyCalls: 10,
   createdAt: new Date(),
 });
+
+export const getMockAuthData = (organizationId?: TenantId): AuthData => ({
```

Comment:
I also prefer this function being here, but is this actually used?
We should remove the other definitions inside the services and use this one